src/venlyController.ts

```diff
@@ -102,11 +110,16 @@ export class VenlyController {
   }
 
   async processPersonalMessage(params: any, req: any) {
+    let message = params.data;
+    try {
+      message = hexToUtf8(params.data);
+    }
+    catch {}
```

Comment:
Would it make sense to log the error if it fails here?

Also I see that before it wasn't converting from hex->utf8. Are we sure this doesn't break anything?